### ISSUE TITLE
DSOAL, sounds play in all channels and vibration restoration

### ISSUE THREAD

**Issue by NoUserNameForYou** [elishacloud/Silent-Hill-2-Enhancements#336]
Hello,

Does this game have ACTUAL positional audio? Tried with both your included alsoft and my edited one. (tried them both in appdata folder and ingame folder) but audio seems like replicated to back channels.

See my example file:

"[General]
channels=surround51rear
frequency=44100
resampler=bsinc24
stereo-mode=headphones
hrtf-mode=full
hrtf=true
default-hrtf=irc_1037_44100
cf_level=0
sources=512
slots=64
sends=16
sample-type=float32
drivers="-dsound,"
period_size=1024
periods=3

[reverb]
boost=0

[decoder]
quad=C:/Users/Administrator/AppData/Roaming/OpenAL/presets/square.ambdec
surround51=C:/Users/Administrator/AppData/Roaming/OpenAL/presets/itu5.1.ambdec
surround71=C:/Users/Administrator/AppData/Roaming/OpenAL/presets/hexagon.ambdec
hq-mode=true"

Now I checked on sensun and it says the game doesn't have LFE, so I also tried itu5.1-nocenter as well as disabling center/LFE and swapping channels etc, with many combinations. When the monsters are behind me, shouldn't I be hearing them from the back channels? Instead, sounds play in all channels. 

---

About vibration:

Can your vibro restoration feature be added to Splinter Cell 1+2+3 and some other same-era games/xbox ports which had directinput?

**Comment by Polymega** [comment]
The finer workings of DSOAL aren't things we're very familiar with, as it's created and maintained by a different party. You can try this configuration for the `alsoft.ini` file and see if it yields the results you're after:

```ini
[General]
channels=
frequency=44100
stereo-mode=auto
cf_level=0
sources=512
slots=64
sends=16
sample-type=float32
resampler=cubic
default-hrtf=Built-In 44100hz
hrtf=auto
period_size=128
periods=3
hrtf-mode=full
hrtf-size=0
```
Beyond that, I'm not sure. Also note that in order for Silent Hill 2 PC to have its speaker configuration setting updated, you yourself must change/update this yourself within Windows' sound control panel.

**Comment by NoUserNameForYou** [comment]
"this configuration for the alsoft.ini file and see if it yields the results you're after:"

Isn't that the default config? 

"ou yourself must change/update this within Windows' sound control panel."

I already said: "disabling center/LFE and swapping channels etc, with many combinations." 

This issue is still open (or the original game's surround sound is broken)

**Comment by Polymega** [comment]
Through the use of DSOAL as-is with our project, it creates 3D positional sound via a binaural presentation.

Beyond that, I'd recommend asking the people over at the "I Drink Lava - 3D Audio" Discord server: https://discord.gg/85hTk6Xpcc

They would be more familiar with DSOAL and how it can be configured to affect SH2 PC, along with how, if at all, it can change the way the game presents 3D positional audio.

**Comment by NoUserNameForYou** [comment]
Thank you for at least trying to help, while I'll skip using that uncouth webservice, your reply is appreciated since there may be people in the same situation.

Also, afaik, DSOAL not only does binaural audio but also restores the surround (very recently played half life 1 cd version with positional 3d audio)

**Comment by Polymega** [comment]
If someone wanted to, they can get very serious with DSOAL and its configuration options--well beyond most of our understanding on it here at this project. 

When we created the alsoft.ini file for DSOAL with our project, we went to that Discord and asked the group what the best "universal settings" would be (the idea being to make the INI as broad/generic as possible to suit most PCs, so the common user wouldn't have to change anything).

This is a long way of saying, since DSOAL is a third party software for our project, we're largely unfamiliar with how/what to change in it for customized a auditory experience. Audio and DSOAL enthusiasts would be the way to go to get proper recommendations.
